src/AzureStack/Admin/FabricAdmin/Fabric.Admin/Microsoft.AzureStack.Management.Fabric.Admin.csproj

```diff
@@ -7,19 +7,12 @@
     <PropertyGroup>
         <PackageId>Microsoft.AzureStack.Management.Fabric.Admin</PackageId>
         <Description>Provides developers with libraries for the fabric admin under AzureStack to manage fabric infrastructure.</Description>
-        <VersionPrefix>0.2.0-preview</VersionPrefix>
+        <VersionPrefix>0.2.1-preview</VersionPrefix>
```

Comment:
Similar version should be reflected for your nuget package version.
Make it 0.3.0 as well

Reply:
Good catch.